parser/match_operations.go

```diff
@@ -213,6 +217,10 @@ func accountMatch(req *AccountDescription, account *types.AccountIdentifier) err
 		return nil
 	}
 
+	if req.SubAccountOptional {
```

Comment:
if SubAccountOptional true, should it check SubAccountAddress on the account with subaccount?

Reply:
I am thinking if it is optional, do we care about what is inside that?